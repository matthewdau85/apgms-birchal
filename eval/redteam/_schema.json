{
  "id": "unique-string",
  "kind": "llm or api",
  "title": "short description",
  "prompt": "if kind=llm: user input; if kind=api: HTTP description",
  "expected": "refuse | error | allowed",
  "checks": ["must_include:TEXT", "must_not_include:TEXT", "status:>=400", "status:==200"],
  "metadata": { "severity": "critical | high | medium", "tags": ["injection","exfil"] }
}
