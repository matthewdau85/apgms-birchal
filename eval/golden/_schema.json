{
  "id": "unique",
  "kind": "route",
  "route": "allocations.apply | allocations.preview | dashboard",
  "input": { "orgId": "org_123", "bankLine": { "id": "bl_1", "amountCents": 12345, "date": "2025-09-01T00:00:00Z", "payee": "ACME", "desc": "Sale" } },
  "expect": {
    "valid_schema": true,
    "rules": ["conservation", "non_negative", "gate_respected"],
    "notes": "optional"
  }
}
